default_config.json

```diff
@@ -2,8 +2,8 @@
 	"name": "lisky",
 	"json": false,
 	"api": {
-		"testnet": false,
-		"node": ""
+		"node": "",
```

Comment:
Should we actually be making this an array of strings?

Reply:
nodes will be the best i think, but should we do it with this PR?

Reply:
https://github.com/LiskHQ/lisky/issues/481